### PR TITLE
[Connector][Doc] Add sqlServer cdc connector doc

### PR DESCRIPTION
**Pull request by ic4y**
## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/source/SqlServer-CDC.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    table-name = "dbo.full_types"
+  }
+}
+```
+
+## Changelog
+
+### next version
+
+- Add SqlServer CDC Source Connector
```
Comment (TaoZex) [inline docs/en/connector-v2/source/SqlServer-CDC.md]: Please add the pr link.

File: docs/en/connector-v2/source/SqlServer-CDC.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+### snapshot.split.size [Integer]
+
+The split size (number of rows) of table snapshot, captured tables are split into multiple splits when read the snapshot
+of table.
+
+### snapshot.fetch.size [Integer]
+
+The maximum fetch size for per poll when read table snapshot.
+
+
```
Comment (TaoZex) [inline docs/en/connector-v2/source/SqlServer-CDC.md]: It is better to delete this blank line.

File: docs/en/connector-v2/source/SqlServer-CDC.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+### database-name [String]
+
+Database name of the database to monitor.
+
+### table-name [String]
+
+Table name is a combination of schema name and table name (schemaName.tableName).
+
+
```
Comment (TaoZex) [inline docs/en/connector-v2/source/SqlServer-CDC.md]: It is better to delete this blank line.

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by EricJoy2048 — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**